docs/bot-usage.md

```diff
@@ -189,6 +190,14 @@ checked before the default locations (The passed path must be a directory!):
 freqtrade trade --strategy AwesomeStrategy --strategy-path /some/directory
 ```
 
+### How to use **--strategy-params**?
+
+This parameter allows you to add parameters of your strategy:
+
+```bash
+freqtrade trade --strategy AwesomeStrategy --strategy-params "{'param1': 5, 'param2': 'something'}"
+```
```

Comment:
Thanks for your quick feedback.

Yes I realized now that it is already possible over config.json, but not over CLI.

The reason why I choosed a JSON representation is because I can just copy the results of the hyperopt runs. I dont think that is to complicated. The parsefun in de Configuration utility takes care  about the conversion from ' to ".

What I really want is the possiblity to pass the strategy params in a separeted way from the config.json. Because I run multiple bots on different exchanges. So the config.json is different for each bot, but the strategy params are all the same and it take some time to synchronize these parameters.

So my ideal use case would be to have an additional json file, only for the strategy params. That's the reason, why i want to pass it in CLI. To be independent.

If you agree I would extend the PR and add an additional config-strategy-params.json file and maybe remove possibility to pass it over CLI.

As a workaround I would handle this file for myself, but I think there has to be a more general way to do that.

Reply:
I know almost noone reads the docs nowadays - but before spending the time implementing a solution, i would still recommend doing that, because what you need can be EASILY accomplished by using multiple `--config` flags - which will be combined (basically, the last key wins).

So you can have your bot specific config - and the startegy params later.
` --config botconfigA.json --config CoolStrategyParams`

[t's also documented](https://www.freqtrade.io/en/latest/bot-usage/#how-to-use-multiple-configuration-files) ?

Reply:
Totally agree with you. That works great. Sorry for wasting your time :( I will close this PR.